docs/components/using-message-components.mdx

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_label: Using Message Components
+---
+
+# Using Message Components
+
+## Overview 
+
+Message components are a powerful way to add interactivity to your messages. They allow you to create rich, interactive experiences for your users, making it easier for them to engage with your content.
+
+### Prerequisites
+
+- You must have a Discord account and be a member of the Discord Developer Portal.
+- You must have a Discord application created in the Discord Developer Portal.
+- You must have the necessary permissions to send messages in the channel where you want to use components.
```

Comment:
I feel like these are a given if you are already looking at the docs. Could be cut in my opinion.